style.css

```diff
@@ -0,0 +1,635 @@
+  position: relative;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.intro-text h1 {
+  font-size: 50px;
+  color: #ec5242;
+  font-weight: 900;
+  word-spacing: 10px;
+}
+
+.p-text {
+  width: 100%;
+  max-width: 600px;
+  padding-left: 20px;
+  flex-wrap: wrap;
+  margin: 40px 0;
+  border: 2px solid #fff;
+  background-color: #d3d3d334;
+}
+
+.intro-date {
+  font-weight: 900;
+  color: #272a31;
+}
+
+.main-program {
+  background-color: #272a31;
+  position: relative;
+  top: 0;
+}
+
+.section-heading {
+  color: #d3d3d3;
+  padding-top: 25px;
+  font-size: 30px;
+  width: 300px;
+  margin: 0 auto;
+}
+
+.section-hr {
+  width: 50px;
+  border: 1px solid;
+  margin: 10px auto;
+  color: #ec5242;
+  margin-bottom: 40px;
+}
+
+.main-program h4 {
+  color: #ec5242;
+}
+
+.program-card-container {
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.program-card {
+  background-color: #d3d3d334;
+  color: #d3d3d3;
+  max-width: 200px;
+  padding: 8px;
+  margin: 5px;
+}
+
+.program-card-icon {
+  font-size: 40px;
+  margin-bottom: 10px;
+}
+
+.program-footer {
+  width: 400px;
+  text-decoration: underline;
+  margin: 0 auto;
+  padding: 50px 0;
+  word-spacing: 7px;
+}
+
+.about-p-text {
+  padding: 15px 50px;
+  flex-wrap: wrap;
+  margin: 40px auto;
+  border: 1px solid rgb(199, 195, 195);
+  background-color: rgba(255, 255, 255, 0.842);
+}
+
+.hackjos {
+  margin-bottom: 50px;
+}
+
+.hackjos-text {
+  max-width: 600px;
+  margin: 0 auto;
+  padding: 50px 0;
+}
+
+.hackjos-text h1 {
+  font-size: 25px;
+  font-weight: 500;
+  margin-bottom: 10px;
+}
+
+.hackjos-logo {
+  height: 200px;
+  max-width: 400px;
+  margin: 0 auto;
+  border: 1px solid rgb(206, 204, 204);
+  padding: 40px;
+  filter: grayscale(50%);
+}
+
+.hackjos-logo img {
+  width: 100%;
+  height: 100%;
+}
+
+.hackjos-logo-text {
+  text-align: center;
+  max-width: 400px;
+  padding: 20px 50px;
+  border: 1px solid rgb(199, 195, 195);
+  background-color: rgba(255, 255, 255, 0.842);
+  margin: 0 auto;
+}
+
+.past-hackjos {
+  margin-top: 50px;
+}
+
+.hackathon-text h1 {
+  font-size: 25px;
+}
+
+.hackathons {
+  justify-content: center;
+}
+
+.hackathons h3,
+p {
+  font-weight: 700;
+}
+
+.hackathon1 h3 {
+  color: #fff;
+}
+
+.hackathon1 p {
+  color: #fff;
+}
+
+.hackathon2 h3 {
+  color: #fff;
+}
+
+.hackathon2 p {
+  color: #fff;
+}
+
+.hackathon1 {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 20px 5px;
+  padding: 50px 30px;
+  background-color: rgba(255, 68, 0, 0.507);
+}
+
+.hackathon1::before {
+  content: '';
+  width: 100%;
+  max-width: 400px;
+  position: absolute;
+  background-image: url('./images/hackjos1.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: -1;
+  opacity: 0.4;
+}
+
+.hackathon2 {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 20px 5px;
+  padding: 50px 30px;
+  background-color: rgba(255, 68, 0, 0.507);
+}
```

Comment:
- [ ] The bottom section images of the about page on desktop view are organized in a horizontal row and should be aligned well. It is good that you organized them in a horizontal raw but, they are not well aligned. I think you need to make the heights of the background images equal so that the alignment is maintained. :pray: 

![image](https://user-images.githubusercontent.com/64894927/132694119-491e49fe-73f1-4294-af40-499db1ff1039.png)
![image](https://user-images.githubusercontent.com/64894927/132694273-bfab22f6-3027-437b-8fa6-a9850804f9cf.png)